src/App.css

```
@@ -1,3 +1,32 @@
 .App {
   text-align: center;
 }
+
+.input {
+  margin-right: 10px;
+  border: 1px solid grey;
+  border-radius: 5px;
+}
+
+.todo {
+  width: 350px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+}
+
+.todoList {
```

мы проходили BEM
мы так называли классы?

```suggestion
.todo-list {
```